edx-platform/pearson-certiport-theme/lms/templates/widgets/footer_language-selector.html

```diff
@@ -0,0 +1,75 @@
+## Language-selection widget for the footer.
```

Comment:
Is this template necessary? @sergivalero20

Reply:
@Squirrel18 It's not being used it right now. So, I''l remove it.